src/TokenParser/SpacelessTokenParser.php

```diff
@@ -24,7 +24,7 @@
  *   {% endspaceless %}
  *   {# output will be <div><strong>foo</strong></div> #}
  *
- * @deprecated since Twig 2.7, to be removed in 3.0 (use the spaceless filter instead)
+ * @deprecated since Twig 2.7, to be removed in 3.0 (use the "apply" tag instead)
```

Comment:
Here, it did not say to use the filter tag, but the spaceless filter. Maybe we should be more precise.
What about `use the "spaceless" filter with the "apply" tag instead`?

Reply:
I'm open to any verbiage change.  I'll push that up.